src/main/java/duke/Pikachu.java

```diff
@@ -0,0 +1,247 @@
+package duke;
+import java.io.*;
+import java.time.*;
+import java.util.*;
+
+import duke.UI.UI;
+import duke.tasks.*;
+
+
+public class Pikachu {
+    private ArrayList<Task> inputList;
+
+    //Constructor
+    public Pikachu() {
+        inputList = new ArrayList<Task>();
+    }
+
+    //Constructor for existing tasklist
+    public Pikachu(ArrayList<Task> inputList) {
+        this.inputList = inputList;
+    }
+
+    //Accessor
+    public ArrayList<Task> getInputList() {
+        return this.inputList;
+    }
+
+    /**
+     * Parses input from the user. This is the main command for Pikachu.
+     *
+     * @param str Input from user.
+     */
+    public void parseInput(String str) {
+        String[] split = str.split(" ");
+
+        /**
+         * "commands"
+         * Prints list of available commands.
+         * Syntax: commands
+         */
```

Comment:
I feel that Javadocs should not be in a method body. Perhaps consider changing to comment blocks instead.
```suggestion
        /*
         "commands"
         Prints list of available commands.
         Syntax: commands
         */
```